server/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java

```diff
@@ -345,6 +345,13 @@ private static void throwEOF(ObjectMapper mapper, String currentFieldName) {
     }
 
     private static void nested(DocumentParserContext context, NestedObjectMapper nested) {
```

Comment:
This method name does not suggest at all that this method is about adding fields to parent/root doc. Should this be renamed to make the purpose clearer?

Reply:
++ have renamed to `copyNestedFields`